src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -491,7 +491,7 @@ public function getWriteInfo(string $class, string $property, array $context = [
         if ($allowMagicCall) {
             [$accessible, $methodAccessibleErrors] = $this->isMethodAccessible($reflClass, '__call', 2);
             if ($accessible) {
-                return new PropertyWriteInfo(PropertyWriteInfo::TYPE_METHOD, 'set'.$camelized, PropertyWriteInfo::VISIBILITY_PUBLIC, false);
+                return new PropertyWriteInfo(PropertyWriteInfo::TYPE_METHOD, 'set'.$camelized, PropertyWriteInfo::VISIBILITY_PUBLIC, false, false);
```

Comment:
Not every `PropertyWriteInfo` and `PropertyReadInfo` instantiations have been updated. Is it intended?

Reply:
I've updated every instantiations where it made sense (i.e: this property has no meaning on a PropertyWriteInfo::TYPE_METHOD ; I even removed this one)